### PR TITLE
ACCSEC-19995 Move xcconfig variables

### PR DESCRIPTION
**Pull request by SergioFierro**
## Jira Ticket
- ACCSEC- 19995

## Description
- Move config variables to a swift class

## Commit message
-  fix: Read SDK version and config

### CODE REVIEW

**Comment by AuthyApps** [comment]
## Code coverage
### Total coverage: **`95.38%`**
File | Coverage
-----|-----
TwilioVerify/TwilioVerify/Sources/Domain/Factor/PushFactory.swift | **`99.33%`**
TwilioVerify/TwilioVerify/Sources/Networking/RequestHelper.swift | **`100.00%`**
> Powered by [Slather](https://github.com/SlatherOrg/slather)

<p align="right" data-meta="generated_by_UnitTests">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by AuthyApps** [comment]
## Code coverage
### Total coverage: **`95.38%`**
File | Coverage
-----|-----
TwilioVerify/TwilioVerify/Sources/Domain/Factor/PushFactory.swift | **`99.33%`**
TwilioVerify/TwilioVerify/Sources/Networking/RequestHelper.swift | **`100.00%`**
> Powered by [Slather](https://github.com/SlatherOrg/slather)

<p align="right" data-meta="generated_by_UnitTests">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by yafuquen** [comment]
Added commit message

File: TwilioVerify/Info.plist [hunk 1]
```diff
@@ -2,27 +2,15 @@
...
 	<string>$(EXECUTABLE_NAME)</string>
 	<key>CFBundleIdentifier</key>
 	<string>$(PRODUCT_BUNDLE_IDENTIFIER)</string>
 	<key>CFBundleInfoDictionaryVersion</key>
 	<string>6.0</string>
-	<key>CFBundleName</key>
-	<string>$(SDK_BUNDLE_NAME)</string>
 	<key>CFBundlePackageType</key>
 	<string>$(PRODUCT_BUNDLE_PACKAGE_TYPE)</string>
-	<key>CFBundleShortVersionString</key>
```
Comment (yafuquen) [inline TwilioVerify/Info.plist:21]: We should keep this to show the version in the SDK's general configuration

**Review by yafuquen — CHANGES_REQUESTED** [review]

**Comment by AuthyApps** [comment]
## Code coverage
### Total coverage: **`95.38%`**
File | Coverage
-----|-----
TwilioVerify/TwilioVerify/Sources/Domain/Factor/PushFactory.swift | **`99.33%`**
TwilioVerify/TwilioVerify/Sources/Networking/RequestHelper.swift | **`100.00%`**
> Powered by [Slather](https://github.com/SlatherOrg/slather)

<p align="right" data-meta="generated_by_UnitTests">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Review by yafuquen — APPROVED** [review]

**Review by Greshot — APPROVED** [review]

**Pull request merged**